lighthouse-core/gather/computed/computed-artifact.js

```diff
@@ -5,13 +5,15 @@
  */
 'use strict';
 
+const CustomMap = require('../../lib/custom-map');
+
 class ComputedArtifact {
   /**
    * @param {!ComputedArtifacts} allComputedArtifacts
    */
   constructor(allComputedArtifacts) {
     /** @private {!Map} */
-    this._cache = new Map();
+    this._cache = new CustomMap(CustomMap.deepEquals);
```

Comment:
instead of passing it into the constructor i'm thinking it might be helpful to clarify the intent of the arg.   

how about:

```js
this._cache = new CustomMap();
this._cache.setEqualityFn(CustomMap.deepEquals);
```

without it being set, it's just using `a === b`

Reply:
good plan 👍